packages/quantic/force-app/examples/main/lwc/configurator/configurator.js

```diff
@@ -12,25 +12,42 @@ export default class Configurator extends LightningElement {
             label: option.label ?? option.attribute,
             description: option.description ?? '',
             defaultValue: option.defaultValue,
+            defaultValueAsText: this.formatDefaultValue(option.defaultValue),
             cyId: `cfg-${option.attribute}`,
         }));
     }
 
+    formatDefaultValue(value) {
+        if (typeof value === 'boolean' || typeof value === 'number' || typeof value === 'string') {
+            return value.toString();
+        }
+
+        return '';
+    }
+
     connectedCallback() {
         this.options.forEach(this.validateOption);
 
         this.options.forEach((option) => {
-            if (option.defaultValue) {
+            if (typeof (option.defaultValue) !== 'undefined') {
```

Comment:
We want to assign the value in case of a null ?

Reply:
You are right, I see no reasons to assign `null` there.